manage_projects.rb

```diff
@@ -31,7 +31,15 @@
 def add_or_update_project(action=nil)
   @factory = ProjectFactory.new
   if action == nil
-    print "List, add or update project(s) (list/add/update/validate)? "
+    print <<-ACTIONS
```

Comment:
Personally I find this a bit confusing, having the list of possible actions before the request to make a choice. And if you enter one of the items exactly as displayed (e.g. `- list`) this is invalid.